docs/src/pages/customization/css-in-js.md

```diff
@@ -155,6 +155,22 @@ function App() {
 export default App;
 ```
 
+## Plugins
+
+JSS uses the concept of plugins to extend his core.
```

Comment:
Ah, sorry, I improved the change, but you beat me to it with the previous one. You move too quickly for me! 😄